source/jquery.flot.time.js

```diff
@@ -37,8 +37,7 @@ API.txt for details.
             microEpoch = Math.round(microEpoch*1000)/1000;
 
             // Microseconds are stored as integers
-            var seconds = microEpoch/1000;
-            this.microseconds = 1000000 * (seconds - Math.floor(seconds));
+            this.microseconds = 1000 * (microEpoch - Math.floor(microEpoch));
```

Comment:
This is the change that fixes issue #1707. We're no longer keeping the milliseconds in the `microseconds` field.